include/tins/ip_reassembler.h

```diff
@@ -190,7 +190,6 @@ class TINS_API IPv4Reassembler {
     address_pair make_address_pair(IPv4Address addr1, IPv4Address addr2) const;
     
     streams_type streams_;
-    OverlappingTechnique technique_;
```

Comment:
Sure thing, no problem! When you see stuff like that either it's currently growing or something that's been deprecated; I made the wrong assumption :smiley: . Done.